VirtoCommerce.CatalogModule.Data/Repositories/CatalogRepositoryImpl.cs

```diff
@@ -691,6 +666,41 @@ public dataModel.PropertyDictionaryItemEntity[] GetPropertyDictionaryItemsByIds(
         }
         #endregion
 
+        protected virtual void LoadVariationsByParentIds(IEnumerable<string> parentIds, ItemResponseGroup respGroup)
```

Comment:
Please change the type of `parentId`s to Array instead of IEnumerable or convert this formal parameter to Array before using in `x => parentIds.Contains(x.ParentId))`  because this code might fail if it will passed not an array as a parameter.